lightweight-term-count-update.php

```diff
@@ -83,9 +83,9 @@ public function setup() {
 		add_action( 'added_term_relationship', array( $this, 'added_term_relationship' ), 10, 3 );
 		add_action( 'deleted_term_relationships', array( $this, 'deleted_term_relationships' ), 10, 3 );
 
-		 /**
-		  * Possibly recount posts for a term once it's been edited.
-		  */
+		/**
```

Comment:
Why is this change part of the PR?

Reply:
This was a PHPCS warning. I needed to trigger a rebuild with the updated .travis.yml file. So I added a commit resolving this PHPCS warning which I found while fixing PHPCS errors in the previous commit. Should I have created a new PR request instead?